lua/autolist/auto.lua

```diff
@@ -175,6 +165,12 @@ function M.new(motion, mapping)
 		return
 	end
 
+	-- check if Treesitter parser is installed, and if so, check if we're in a markdown code fence
+	local parser = require('autolist.treesitter'):new(vim.api.nvim_get_current_buf(), vim.api.nvim_get_current_win())
```

Comment:
This would be run every time `new` is run right? I think this could be moved to the top of the file so it doesn't get run every time. Sorry for the contradictory messages.